kermit/src/androidMain/kotlin/co/touchlab/kermit/JvmMutableLoggerConfig.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021 Touchlab
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+package co.touchlab.kermit
+
+import kotlin.jvm.Volatile
+
+internal class JvmMutableLoggerConfig : MutableLoggerConfig {
+    @Volatile
+    private var _minSeverity: Severity = DEFAULT_MIN_SEVERITY
+
+    @Volatile
+    private var _loggerList: List<LogWriter> = listOf(
+        CommonWriter()
+    )
+
+    @Volatile
+    private var _defaultTag: String = "Kermit"
+
+    override var minSeverity: Severity
+        get() = _minSeverity
+        set(value) {
+            synchronized(this) {
+                _minSeverity = value
+            }
+        }
+
+    override var loggerList: List<LogWriter>
```

Comment:
This is failing, though. The field was renamed to `logWriterList`. I assume that's what's happening?

Reply:
fixed